src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -291,14 +162,14 @@ class VanillaUmiConsensusCaller
     *
     * The same number of base sequences and quality sequences should be given.
     * */
-  private[umi] def consensusCall(reads: Seq[SourceRead]): Option[ConsensusRead] = {
+  private[umi] def consensusCall(reads: Seq[SourceRead]): Option[VanillaConsensusRead] = {
     // check to see if we have enough reads.
     if (reads.size < this.options.minReads) {
```

Comment:
It's a little belt-and-suspenders, but it is necessary.  The order of operations is:
1. Check that we have enough reads on 141
2. Filter to the most common alignment
3. Map reads to source reads and call this method
4. Check that we have enough reads left after filtering.

I'll move the check up to immediately after filtering and (so long as it doesn't break any tests) remove it from here.